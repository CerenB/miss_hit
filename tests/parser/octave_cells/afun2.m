% Extracted from the Octave tests parser.tst
function afun2()
    a = {1, @sin, 2, @cos};
    b = {1 @sin 2 @cos};
    assert (a == b);
end
