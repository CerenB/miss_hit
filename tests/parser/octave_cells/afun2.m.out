----------------------------------------------------------------------
--  Parse tree for afun2.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <afun2>
    Body: Sequence_Of_Statements
      Statements: Simple_Assignment_Statement
        LHS: Identifier <a>
        RHS: Cell_Expression
          Rows: Row
            Items: Number_Literal <1>
            Items: Function_Pointer
              Name: Identifier <sin>
            Items: Number_Literal <2>
            Items: Function_Pointer
              Name: Identifier <cos>
      Statements: Simple_Assignment_Statement
        LHS: Identifier <b>
        RHS: Cell_Expression
          Rows: Row
            Items: Number_Literal <1>
            Items: Function_Pointer
              Name: Identifier <sin>
            Items: Number_Literal <2>
            Items: Function_Pointer
              Name: Identifier <cos>
      Statements: Naked_Expression_Statement
        Expression: Reference
          Name: Identifier <assert>
          Arguments: Binary_Operation ==
            LHS: Identifier <a>
            RHS: Identifier <b>
----------------------------------------------------------------------
In afun2.m, line 3
|     a = {1, @sin, 2, @cos};
|           ^ info: this token is not linked to the ast
In afun2.m, line 3
|     a = {1, @sin, 2, @cos};
|                 ^ info: this token is not linked to the ast
In afun2.m, line 3
|     a = {1, @sin, 2, @cos};
|                    ^ info: this token is not linked to the ast
In afun2.m, line 4
|     b = {1<anon,> @sin 2 @cos};
|           ^^^^^^^ info: this token is not linked to the ast
In afun2.m, line 4
|     b = {1 @sin<anon,> 2 @cos};
|                ^^^^^^^ info: this token is not linked to the ast
In afun2.m, line 4
|     b = {1 @sin 2<anon,> @cos};
|                  ^^^^^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
